redbot/core/utils/chat_formatting.py

```diff
@@ -284,7 +284,7 @@ def pagify(
             yield in_text
 
 
-def rows_to_embeds(
+def embeds_from_rows(
```

Comment:
```suggestion
def embedify_list(
```
Alternative name option. Follows the naming scheme of `pagify`